receiver/prometheusremotewritereceiver/config.go

```diff
@@ -0,0 +1,18 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package prometheusremotewritereceiver // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/prometheusremotewritereceiver"
+
+import (
+	"go.opentelemetry.io/collector/component"
+)
+
+// Config holds common fields and embedded protocol-specific configurations
+type Config struct{}
```

Comment:
Shouldn't the first PR include the component's specific configuration?

https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#adding-new-components

Reply:
I see! TBH I'm not sure if that's a strong requirement. I'm fine either way if there is no other objection!